### ISSUE TITLE
Flake in kubectl e2e test

### ISSUE THREAD

**Issue by miminar** [openshift/origin#8397]
• Failure [337.093 seconds]
    Kubectl client
    /data/src/github.com/openshift/origin/Godeps/_workspace/src/k8s.io/kubernetes/test/e2e/kubectl.go:1146
      Update Demo
      /data/src/github.com/openshift/origin/Godeps/_workspace/src/k8s.io/kubernetes/test/e2e/kubectl.go:161
        should scale a replication controller [Conformance] [It]
        /data/src/github.com/openshift/origin/Godeps/_workspace/src/k8s.io/kubernetes/test/e2e/kubectl.go:150

        Apr  7 04:48:41.087: Timed out after 300 seconds waiting for name=update-demo pods to reach valid state

        /data/src/github.com/openshift/origin/Godeps/_workspace/src/k8s.io/kubernetes/test/e2e/util.go:1469

Occured in [job 14150](https://ci.openshift.redhat.com/jenkins/job/test_pull_requests_origin/14150/consoleFull), PR #7596.

**Comment by ncdc** [comment]
```
Apr  7 04:48:43.612: INFO: At 2016-04-07 04:43:59 -0400 EDT - event for update-demo-nautilus-efsnr: {kubelet 172.18.5.224} FailedSync: Error syncing pod, skipping: failed to "StartContainer" for "update-demo" with RunContainerError: "failed to apply oom-score-adj to container \"open /proc/1349/cgroup: no such file or directory\"- /k8s_update-demo.403aafea_update-demo-nautilus-efsnr_e2e-tests-kubectl-5zw9t_d144e530-fc9c-11e5-871b-0ef7c2e23739_94ae9dfe"
```

Upstream https://github.com/kubernetes/kubernetes/issues/22639 and/or https://github.com/kubernetes/kubernetes/issues/23607

**Comment by pecameron** [comment]
liggitt, not sure of root cause. src/k8s.io/kubernetes/pkg/kubelet/dockertools/manager.go:	return kubecontainer.ContainerID{}, fmt.Errorf("failed to apply oom-score-adj to container %q- %v", err, containerInfo.Name)
I have run [TEST] for PR 8334 many times. The tests end in failure in different places. I am not sure what is going on.

**Comment by ncdc** [comment]
@pecameron your error should be fixed by #8412
